csvcubed-models/csvcubedmodels/rdf/dcat.py

```diff
@@ -63,8 +63,8 @@ class Resource(NewMetadataResource):
         Triple(DCAT.keyword, PropertyStatus.recommended, map_str_to_en_literal),
     ]
     landing_page: Ann[
-        Optional[str], Triple(DCAT.landingPage, PropertyStatus.recommended, URIRef)
-    ]
+        Set[list[str]], Triple(DCAT.landingPage, PropertyStatus.recommended, URIRef)
+    ]  # todo: Enable support for multiple landing pages here, using a Set.
```

Comment:
Make sure you initialise the set in the `__init__` function. See what we're doing with themes/keywords there.